scripts/build_datapackage.py

```diff
@@ -243,6 +245,7 @@ def calculate_emission_limit(
 
     logfile = sys.argv[3]
     logger = config.add_snake_logger(logfile, "build_datapackage")
+    initLogger(logger)
```

Comment:
Could this function be called from within config.add_snake_logger? This would avoid a lot of code duplication in the scripts.